tests/TestCase.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaravelDaily\Invoices\Tests;
+
+use LaravelDaily\Invoices\InvoiceServiceProvider;
+use Orchestra\Testbench\Concerns\WithWorkbench;
+use Orchestra\Testbench\TestCase as BaseTestCase;
+
+abstract class TestCase extends BaseTestCase
```

Comment:
use Orchestra to be able to handle Laravel related stuff, as this is a Laravel package.

Reply:
is it possible to not have additional dependency?

Reply:
I understand where you're coming from, however without this dependency it is not possible to use Laravel features like the translator.

Besides that point, this dependency is the de-facto default dependency in Laravel packages. Or at least in the ones I have seen.
In the end it is up to you whether or not to include this dependency. If you choose not to, then you can not use Laravel related functions, helpers, etc.